pkg/kubelet/kuberuntime/helpers.go

```diff
@@ -39,6 +41,9 @@ const (
 	// 100000 is equivalent to 100ms
 	quotaPeriod    = 100 * minQuotaPeriod
 	minQuotaPeriod = 1000
+
+	// Temporary annotation until API review https://pr.k8s.io/58716 is complete
+	shareProcessAnnotation = "kubelet.alpha.kubernetes.io/share-process-namespace"
```

Comment:
If there is a possibility for us to use the annotation for the feature, it should be moved to `pkg/apis/core/annotation_key_constants.go` with proper naming documentation.

If this is not going to be used, I think there is no need to add a temporary annotation...

Reply:
Ok, I think we can get the API change in time, so I'll remove the annotation. Let's merge the rest of this PR without it and I'll follow up after (or add it to #58716 if the timing works out). PTAL.